test-cases/performance/perf-search-best-throughput-config.yaml

```diff
@@ -32,3 +32,5 @@ email_recipients: ['scylla-perf-results@scylladb.com']
 store_perf_results: false
 # tests are running on aws. The loader uses the shard-aware driver
 ami_id_loader: 'scylla-qa-loader-ami-v19'
+
+client_encrypt: true
```

Comment:
@aleksbykov I hoped to controlled it from the pipeline.
By default it should be probably set to false, but when checking it, set to true.

Reply:
Ok. i will redo